app/ante/ante.go

```diff
@@ -56,6 +56,7 @@ func NewAnteHandler(ak auth.AccountKeeper, evmKeeper EVMKeeper, sk types.SupplyK
 			anteHandler = sdk.ChainAnteDecorators(
 				NewEthSetupContextDecorator(), // outermost AnteDecorator. EthSetUpContext must be called first
 				NewEthMempoolFeeDecorator(evmKeeper),
+				NewGasLimitDecorator(evmKeeper),
```

Comment:
This ante handler can be placed at the top

Reply:
done